Directory.Build.props

```
@@ -1,5 +1,41 @@
 <Project>
-  <Import Project="eng\Versions.props" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup> 
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType>portable</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <LangVersion>Latest</LangVersion>
+    <IsShipping>true</IsShipping>
+    <SignAssembly>false</SignAssembly>
+    <MauiRootDirectory>$(MSBuildThisFileDirectory)</MauiRootDirectory>
+    <MauiSrcDirectory>$(MSBuildThisFileDirectory)src/</MauiSrcDirectory>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- NuGet package information -->
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <Company>Microsoft</Company>
+    <Product>Microsoft MAUI</Product>
```

```suggestion
    <Product>Microsoft .NET MAUI</Product>
```

Do we have an official name?

---

Ok I took this, is correct right @davidortinau ? I also fixed [our unit test for this ](https://github.com/dotnet/maui/pull/21487/commits/bfec68b484858bb67a784b49e14fcae85c95c73b)